mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -4,6 +4,10 @@ CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
 	COMMENT Biome placement starts here, however all vanilla and most modded chunk generators delegate this to a {@linkplain net.minecraft.world.biome.source.BiomeSource biome source}.
 	FIELD field_12761 biomeSource Lnet/minecraft/class_1966;
 	FIELD field_16567 config Lnet/minecraft/class_5311;
+	FIELD field_24748 worldSeed_ J
```

Comment:
Underscore sucks, but while it's private, Enigma won't let me name it the same as a field in the subclass.

Reply:
```suggestion
	FIELD field_24748 worldSeed J
```
yeah you can do this hack here